subseas_flex_fcst/maproom.py

```diff
@@ -422,7 +468,8 @@ def fcst_tiles(tz, tx, ty, proba, variable, percentile, threshold, start_date,le
         dims = fcst_mu.rename({"X": "lon", "Y": "lat"}).dims
     # pingrid.tile wants 2D data
     ).squeeze("T")
-    fcst_cdf = fcst_cdf.squeeze("S")
+    if "S" in fcst_cdf.dims:
+        fcst_cdf = fcst_cdf.squeeze("S")
```

Comment:
Why would the S dim not be present?

Reply:
So... it's a bit messy as of now. Most of the time coords and dims put up in the cpt DataArrays are garbage (note that if they weren't, we probably would have much less work to do in retrieving/formatting dates).

In the subseasonal Bangladesh case, there is no S in the DataArray whatsoever, but Drew's sel_cpt_file adds it (expands_dim), so it needs be squeezed later on. in the seasonal Guatemala case, there is a S coord that is not a dim... so I could not use expands_dim in sel_cpt_file, but I need not squeeze it later on because not a dim.

This is sad. We could go simple and remove the expands_dim of sel_cpt_file and figure curating cleanly those files later. OR we have sel_cpt_file return a DataArray with a correct S value in it.

Reply:
I see (mostly). I agree that the right approach is to modify sel_cpt_file so that the thing it returns always has the same dimensions, despite variability in the shape of the original datasets. I have no opinion on whether you should do that by always providing S or never providing S (for the time being).

I didn't understand the specifics of the BD and Guatemala datasets. Let me know if you need me to look closer.

Reply:
I went for the simplest, ignoring the garbage in cpt DataArray, not trying to correct it, then squeezing everything. We'll have to figure that out at some point, but maybe first figure out if we'll get cpt DataArray that are not garbage on that front and figure out then how to identify the cases and deal with them.